docs/current_docs/quickstart/883939-containers.mdx

```diff
@@ -65,43 +59,46 @@ While most terminal programs such as `htop` or `vim` work with `dagger ... termi
 
 ### Execute commands in the container
 
-The `Container` type has a `WithExec()` function, which returns the container after executing a specific command inside it. So, you could achieve the same result as before (although non-interactively) by chaining a function call to `withExec()` on the `Container` returned previously, as shown in the following command:
+The `Container` type has a `WithExec()` function, which returns the container after executing a specific command inside it. So, you could achieve the same result as before (although non-interactively) by chaining function calls to `WithExec()` and `Stdout()` on the `Container` returned previously, as shown in the following command:
 
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.2 call container --packages=cowsay with-exec --args cowsay,dagger stdout
+dagger -m github.com/shykes/daggerverse/wolfi@v0.1.2 call \
+  container --packages=python3 \
+  with-exec --args=python3,-c,"print('Hello from Dagger')" \
+  stdout
+```
+
+You should see the following output:
+
+```shell
+Hello from Dagger
 ```
 
 ### Publish the container image to a registry
 
-The `Container` type also has a `Publish()` function, which publishes the container to a registry. To see this in action, call the previous function again, this time chaining an additional function call to `Publish()` on the returned `Container`:
+The `Container` type also makes a number of other functions available. To see some of these in action, call the previous function again, chaining additional function calls to `WithEntrypoint()` and `Publish()` on the returned `Container` to, respectively, set the container entrypoint command and publish it to a registry:
```

Comment:
`, respectively,` feels excessive, I think removing it makes it easier to read this sentence.

Reply:
Done